### PR TITLE
[Feature] Create `browse_transform.py`

### PR DESCRIPTION
**Pull request by matrixgame2018**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily get feedback. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation

Please describe the motivation for this PR and the goal you want to achieve through this PR.

## Modification

Please briefly describe what modification is made in this PR.

## BC-breaking (Optional)

Does the modification introduce changes that break the backward compatibility of the downstream repos?
If so, please describe how it breaks the compatibility and how the downstream projects should modify their code to keep compatibility with this PR.

## Use cases (Optional)

If this PR introduces a new feature, it is better to list some use cases here and update the documentation.

## Checklist

1. Pre-commit or other linting tools are used to fix potential lint issues.
2. The modification is covered by complete unit tests. If not, please add more unit tests to ensure the correctness.
3. If the modification has a potential influence on downstream projects, this PR should be tested with downstream projects, like MMDetection or MMClassification.
4. The documentation has been modified accordingly, like docstring or example tutorials.

### CODE REVIEW

File: tools/analysis_tools/browse_transform.py [hunk 1]
```diff
@@ -0,0 +1,243 @@
+# Copyright (c) OpenMMLab. All rights reserved.
```
Comment (hhaAndroid) [inline tools/analysis_tools/browse_transform.py:1]: 这个要单独新建一个文件吗？可以继承到 browse-dataset里面吗？

File: tools/analysis_tools/browse_transform.py [hunk 2]
```diff
@@ -0,0 +1,243 @@
...
+from mmengine.config import Config, DictAction
+from mmengine.dataset import Compose
+from mmengine.utils import ProgressBar
+from mmengine.visualization import Visualizer
+from mmyolo.utils import register_all_modules
+from mmyolo.registry import DATASETS, VISUALIZERS
+
+def parse_args():
+    parser = argparse.ArgumentParser(description='Browse a dataset')
+    parser.add_argument('--config',default="/media/airs/Newsmy1/mmyolo/configs/yolov5/yolov5_balloon.py",help='train config file path')
```
Comment (hhaAndroid) [inline tools/analysis_tools/browse_transform.py:17]:
```suggestion
    parser.add_argument('--config',help='train config file path')
```

File: tools/analysis_tools/browse_transform.py (hunk 1 above)
Comment (hhaAndroid) [reply tools/analysis_tools/browse_transform.py:1]: 要提供相应的使用文档和效果图，不然不好 review。先写中文使用文档

File: tools/analysis_tools/browse_transform.py (hunk 2 above)
Comment (PeterH0323) [reply tools/analysis_tools/browse_transform.py:17]: 建议将 --config 改为 config ，和其他脚本一致

**Comment by matrixgame2018** [comment]
我把这个prj继承到原本的browse-dataset

**Pull request closed without merging**